src/styles/components/side-panel-detail.less

```diff
@@ -226,6 +226,23 @@ Side Panel Detail
 
     width: @base-spacing-unit * 7;
 
+    &-wide {
+
+      width: @base-spacing-unit * 12;
```

Comment:
When we create styles like these we need to make sure to create that spacing for all screens using all of the base spacing unit definitions: `@base-spacing-unit`, `@base-spacing-unit-screen-mini`, `@base-spacing-unit-screen-small`, etc.